components/NavBar/DrawerNav.tsx

```diff
@@ -0,0 +1,118 @@
+        }
+      >
+        <div>
+          <div className="flex w-full items-center justify-between">
+            <Link onClick={handleNav} href="">
+              <Image
+                src="/navbar assets/navLogo.png"
+                alt=""
+                width="100"
+                height="50"
+              />
+            </Link>
+            <div
+              onClick={handleNav}
+              className="rounded-full shadow-lg shadow-gray-400 p-3 cursor-pointer"
+            >
+              <AiOutlineClose />
+            </div>
+          </div>
+          <div className="border-b border-gray-300 my-4">
+            <p className="w-[85%] md:[90%] py-4">
+              Lets build something legendary together
+            </p>
+          </div>
+          <div>
+            <ul className="py-4 flex-col">
+              <Link href="">
+                <li onClick={handleNav} className="py-4 text-sm">
+                  {" "}
+                  Home
+                </li>
+              </Link>
+              <Link href="/#about">
+                <li onClick={handleNav} className="py-4 text-sm">
+                  {" "}
+                  About
+                </li>
+              </Link>
+              <Link href="/#skills">
+                <li onClick={handleNav} className="py-4 text-sm">
+                  {" "}
+                  Skills
+                </li>
+              </Link>
+              <Link href="/#projects">
+                <li onClick={handleNav} className="py-4 text-sm">
+                  {" "}
+                  Projects
+                </li>
+              </Link>
+              <Link href="/#contact">
+                <li onClick={handleNav} className="py-4 text-sm">
+                  {" "}
+                  Contact
+                </li>
+              </Link>
+            </ul>
+            <div className="pt-40">
+              <p className="uppercase tracking-widest text-[#5651e5]">
+                Lets Connect
+              </p>
+              <div className="flex items-center justify-between my-4 w-full sm:w[80%] ">
+                <div className="rounded-full shadow-lg shadow-gray-400 p-3 cursor-pointer hover:scale-105 ease-in duration-300">
+                  <Link href="https://www.linkedin.com/in/khaled-hazem-711527223/">
+                    <FaLinkedinIn />
+                  </Link>
+                </div>
+                <div className="rounded-full shadow-lg shadow-gray-400 p-3 cursor-pointer hover:scale-105 ease-in duration-300">
+                  <Link href="https://github.com/KhaledHN">
+                    <FaGithub />
+                  </Link>
+                </div>
+                <div className="rounded-full shadow-lg shadow-gray-400 p-3 cursor-pointer hover:scale-105 ease-in duration-300">
+                  <Link href="https://mail.google.com/mail/u/0/#sent?compose=CllgCJqXxhFbpRhxNlVPhLsSwWrlkqhLtXRKFkCtRfxVBnfFHZrnmZNmzdTStkWKskMNjXHPRhg">
```

Comment:
Check mailto: https://www.w3docs.com/snippets/html/how-to-create-mailto-links.html